eslint.config.mjs

```diff
@@ -611,32 +612,67 @@ export default tseslint.config(
     ],
     rules: {
       '@typescript-eslint/sort-type-constituents': 'off',
-      'perfectionist/sort-classes': [
+      'perfectionist/sort-classes': 'error',
+      'perfectionist/sort-enums': 'off',
+      'perfectionist/sort-objects': 'error',
+      'perfectionist/sort-union-types': [
         'error',
         {
-          order: 'asc',
-          partitionByComment: true,
+          groups: ['unknown', 'keyword', 'nullish'],
           type: 'natural',
         },
       ],
-      'perfectionist/sort-enums': 'off',
+      'simple-import-sort/imports': 'off',
+    },
+    settings: {
+      perfectionist: {
+        partitionByComment: true,
+        order: 'asc',
+        type: 'natural',
+      },
+    },
+  },
+  {
+    files: ['packages/ast-spec/src/**/*.ts'],
+    rules: {
+      'perfectionist/sort-interfaces': [
+        'error',
+        {
+          customGroups: {
+            primary: ['type'],
+          },
+          groups: ['primary', 'unknown'],
+        },
+      ],
+    },
+  },
+  {
+    files: ['packages/eslint-plugin/src/rules/*.ts'],
+    rules: {
       'perfectionist/sort-objects': [
         'error',
         {
-          order: 'asc',
-          partitionByComment: true,
-          type: 'natural',
+          customGroups: {
+            primary: ['loc', 'name', 'node', 'type'],
+            secondary: ['meta', 'messageId', 'start'],
+            tertiary: ['defaultOptions', 'data', 'end'],
+            quaternary: ['fix'],
```

Comment:
You know.... You could have just called them "first" "second" "third" and "fourth" haha

Reply:
lol I liked these